kratos/includes/table_stream.h

```diff
@@ -15,16 +15,23 @@
 #ifndef KRATOS_TABLE_STREAM_H_INCLUDED
```

Comment:
there are some comments in the header of this file, I think at least some of them are outdated

Reply:
:grin: Maybe, but this class will disappear eventually